inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/pulsar/PulsarSourceOperator.java

```diff
@@ -131,8 +131,10 @@ public Map<String, List<StreamSource>> getSourcesMap(InlongGroupInfo groupInfo,
             pulsarSource.setAdminUrl(adminUrl);
             pulsarSource.setServiceUrl(serviceUrl);
             pulsarSource.setInlongComponent(true);
-            String serializationType = DataTypeEnum.forType(streamInfo.getDataType()).getType();
-            pulsarSource.setSerializationType(serializationType);
+            if (StringUtils.isNotBlank(streamInfo.getDataType())) {
```

Comment:
Is dataType not necessary for tube or kafka? If true, please add judgment to avoid the occurrence of null pointers, fault tolerance in TubeMQSourceOperator、KafkaSourceOperator.

Reply:
@chestnut-c If the dataType was blank, do we need a default value for it?

Reply:
> 

We only tested for PulsarSource, and also took the serializationType of StreamSource, so InlongStreamInfo is not a necessary DataType @fuweng11  you can that context.

Reply:
> @chestnut-c If the dataType was blank, do we need a default value for it?



> PulsarSource

I think the current protocol does not need a default value, but PulsarSource takes values ​​from two places, which leads to the need for fault tolerance

Reply:
> > @chestnut-c If the dataType was blank, do we need a default value for it?
> 
> > PulsarSource
> 
> I think the current protocol does not need a default value, but PulsarSource takes values ​​from two places, which leads to the need for fault tolerance
@chestnut-c @healchow 
It have a default value.
```java
 // if the SerializationType is still null, set it to the CSV
            if (StringUtils.isEmpty(pulsarSource.getSerializationType())) {
                pulsarSource.setSerializationType(DataTypeEnum.CSV.getType());
            }
```

Reply:
if the SerializationType is still null, set it to the CSV